setup.py

```diff
@@ -6,8 +6,8 @@
 setuptools.setup(
     name="reinvent_models",
     version="0.0.7",

```

Comment:
is this version correct?

Reply:
I have just pushed an update for this :)